src/components/try-hcp-callout/components/try-hcp-callout-prebuilt/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { TryHcpCallout } from '..'
+import { ProductSlugWithContent } from '../../types'
+import { tryHcpCalloutContent } from './content'
+
+/**
+ * A wrapper around TryHcpCallout.
+ *
+ * Handles content for a set of supported `productSlug` value, which allows
+ * this component to provide a single-prop `productSlug` props API.
+ * Intended for use in authorable contexts.
+ */
+export function TryHcpCalloutPrebuilt({
```

Comment:
This is component that'll be dropped into authorable contexts - it accepts a `productSlug` and that's it.

Reply:
Love it. We should make sure to add this to the authoring documentation for available components. We're adding this to product landing pages and in MDX content?

Reply:
☝️ I think we won't be adding this to MDX, at least not yet! AFAIK the placements are only:
- On the `/<product>` landing pages (authored in `.json`)
- In the sidecar of install views (authored in JSX)
I can definitely add some docs for the `.json` authoring context, great call 💯 I'll do that in the next PR where I implement the authoring, if that sounds good!

Reply:
Oh okay cool! That makes it easier then if its not in MDX. Doing in the next PR sounds great 😄